meinberlin/apps/offlineevents/templates/meinberlin_offlineevents/offlineevent_detail.html

```diff
@@ -20,4 +20,4 @@ <h2 class="item-detail__title">{{ object.name }}</h2>
             </article>
         </div>
     </div>
-{% endblock %}
+{% endblock %} -->
```

Comment:
We do not appear to use this template but have only commented out for now for testing